File: templates/matlab/physics_field_stage_step.m
function fig = physics_field_stage_step()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('step_curve', 2017, 'physics field analysis: stage step curve', 'physics field analysis', 'stage step curve');
end

File: templates/python/physics_field_stage_step.py
"""physics_field_stage_step: 物理场分析阶段阶梯曲线（step 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='physics field analysis: stage step curve'):
    return make_template_figure('step_curve', seed=2017, title=title, domain='physics field analysis', topic='stage step curve')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
